gnosis/safe-react
# Fix: show non-owner execution warning only on tx creation

## What it solves
Non-owner error message should never appear when executing an existing tx.
It can only be shown if a non-owner is trying to create a tx (currently only possible via a Sage App).

## How to test
Case 1:
* Make a tx that would inevitably fail
* Try to execute it as a non-owner
* It should say: "tx will fail"

Case 2:
* As a non-owner, try to create a tx from Drain Safe
* It should say: "not an owner"
